src/utils/weights.rs

```diff
@@ -53,8 +57,8 @@ impl WeightInit {
         Value::from(weight)
     }
 
-    fn he_uniform(&self, fan_num: usize) -> Value {
-        let limit = (3.0 / (fan_num as f64)).sqrt();
+    fn he_uniform(&self, fan_in: usize) -> Value {
```

Comment:
I think this should still be called `fan_num` because, in the Pytorch docs, it says that the fanning could be either the number of input or output connections

Reply:
Demba and I were talking about this. I told him to go ahead and change it to match TensorFlow, because the README links to TF documentation. Interestingly, TF's implementation of Lecun normal is almost identical to that of PyTorch's He normal. Up to you though.

Reply:
Alright, I tried finding any references to fan in or fan out by skimming over the [paper](https://arxiv.org/pdf/1502.01852.pdf) but couldn't find it. Other resources do call it fan in so good fix Demba!